py-polars/pyproject.toml

```
@@ -54,16 +56,24 @@ matplotlib = ["matplotlib"]
 numpy = ["numpy >= 1.16.0, < 2.0.0"]
 openpyxl = ["openpyxl >= 3.0.0"]
 pandas = ["pyarrow >= 7.0.0", "pandas"]
-plot = ["hvplot >= 0.9.1"]
 pyarrow = ["pyarrow >= 7.0.0"]
 pydantic = ["pydantic"]
 sqlalchemy = ["sqlalchemy", "pandas"]
 style = ["great-tables >= 0.8.0"]
-timezone = ["backports.zoneinfo; python_version < '3.9'", "tzdata; platform_system == 'Windows'"]
 xlsx2csv = ["xlsx2csv >= 0.8.0"]
 xlsxwriter = ["xlsxwriter"]
+# ------------------------
+# feature bundles
+# ------------------------
+excel = ["fastexcel", "openpyxl", "xlsx2csv", "xlsxwriter"]
```

I don't think this works. It has to be specified like below, otherwise you don't get the version constraints.
```suggestion
excel = ["polars[fastexcel,openpyxl,xlsx2csv,xlsxwriter]"]
```